content/applications/inventory_and_mrp/inventory/barcode/operations/receipts_deliveries.rst

```diff
@@ -0,0 +1,183 @@
+=============================================
+Process receipts and deliveries with barcodes
+=============================================
+
+The :guilabel:`Barcode app` can be used to process receipts, deliveries, and other types of
+operations in real time using a barcode scanner or the Odoo mobile app.
+
+This makes it possible to process operations on the warehouse floor when they happen, instead of
+having to wait to validate transfers from a computer. Processing operations this way can help to
+properly attribute barcodes to the appropriate products, pickings, locations, and more.
+
+Enable the Barcode app
+======================
+
+To use the :guilabel:`Barcode app` to process transfers, it must be installed by enabling the
+feature from the settings of the :guilabel:`Inventory app`. To do so, go to the
+:menuselection:`Inventory app --> Configuration --> Settings`. Then, scroll down to the
+:guilabel:`Barcode` section, and click the checkbox next to the :guilabel:`Barcode Scanner` feature.
+
+Once the checkbox is ticked, click :guilabel:`Save` at the top of the page to save changes. Once the
+page has refreshed, new options will be displayed under the :guilabel:`Barcode Scanner` feature:
+:guilabel:`Barcode Nomenclature` (with a corresponding drop-down menu), where either
+:guilabel:`Default Nomenclature` or :guilabel:`Default GS1 Nomenclature` can be selected, a
+:guilabel:`Configure Product Barcodes` internal link arrow, and a set of :guilabel:`Print` buttons
+for printing barcode commands and a barcode demo sheet.
+
+.. image:: receipts_deliveries/receipts-deliveries-barcode-setting.png
+   :align: center
+   :alt: Enabled Barcode feature in Inventory app settings.
+
+For more on setting up and configuring the :guilabel:`Barcode` app, refer to the :doc:`Set up your
+barcode scanner </applications/inventory_and_mrp/inventory/barcode/setup/hardware>` and
+:doc:`Activate the Barcodes in Odoo
+</applications/inventory_and_mrp/inventory/barcode/setup/software>` documentation pages.
+
+Scan barcodes for receipts
+==========================
+
+To process warehouse receipts for incoming products, there first needs to be a purchase order (PO)
+created, and a receipt operation to process.
+
+To create a :abbr:`PO (purchase order)`, navigate to the :menuselection:`Purchase app --> Create` to
+create a new request for quotation (RFQ). From the blank :abbr:`RFQ (request for quotation)` form,
+click the drop-down menu next to the :guilabel:`Vendor` field to add a vendor. Then, on the
+:guilabel:`Product` line under the :guilabel:`Products` tab, click :guilabel:`Add a product`, and
+select the desired product(s) to add to the quotation.
```

Comment:
I add in the detailed explanation because I want it to be helpful for people who have 0 knowledge about what they're looking at or how to do it, so I think it's helpful to include